### ISSUE TITLE
A-Jar  " If your project is being revision controlled using Git/GitHub, do not commit the JAR file created."

### ISSUE THREAD

**Issue by baskargopinath** [nus-cs2103-AY2324S2/forum#196]
How do i know if my project is being revision controlled? i searched online and it says if there a .git folder inside our project it means its being revision controlled. is this correct?

**Comment by damithc** [comment]
@bgopi23 Duke project is used by multiple courses, some of which don't teach revision control. Hence the phrase `If your project is being revision controlled using Git/GitHub, ...`

Your iP is using Git and GitHub, so yes, it _is_ being revision controlled.

**Comment by Jerome-j** [comment]
Hi @bgopi23 not sure if I understood your question properly. 

Yes, the presence of a .git folder within your project directory is an indicator that your project is under revision control using Git.

In this case, if you do not specify inside the .gitignore file, every JAR file you create will automatically be tracked by Git - so Prof is trying to tell us not to commit the JAR file as a source code - rather introduce it as a release.

In summary, if your code updates are reflected on GitHub (meaning you can see the changes on GitHub), then your project is under revision control with Git.

**Comment by NeoHW** [comment]
Hi, Git is a distributed version control system that enables us to have multiple local copies of the project's codebase independent of each other. 
In the case of cs2103, we have learnt and followed the forking, committing and execution of other git actions, which shows that our projects are already indeed version controlled

**Comment by baskargopinath** [comment]
i see, thanks @NeoHW @damithc @Jerome-j
